### ISSUE TITLE
BUG: Placeholder hides input

### ISSUE THREAD

**Issue by j-eick** [j-eick/vocab-dojo#29]
### Description

🚨

1. FrontTitle gets input
2. clicking "Create Card"
3. Warning: "Input is missing" (BackInput is missing)
4. Frontside input is visibly gone. User only enters input for backSide and clicks "Create Card. 
5. Card is created
6. Initally entered input in frontside was seemingly gone after step 2, but it was actually still there. Hence after entering backside input and clicking "Create Card", a new card with the previously invisible Frontside input is created. 

🚨
